backend/tilavarauspalvelu/api/graphql/types/recurring_reservation/serializers/deny_serializer.py

```diff
@@ -66,6 +68,15 @@ def update(self, instance: RecurringReservation, validated_data: dict[str, Any])
             handled_at=now,
         )
 
+        # If any reservations had access codes, reschedule the series to remove all denied reservations.
+        # This might remove leave an empty series, which is fine.
+        if has_access_code:
+            # Allow mutation to succeed if Pindora request fails.
```

Comment:
The mutation should fail if the Pindora reschedule fails (except that `NotFound` is OK), since we don't want to leave reservees with functioning access codes (same as denying an individual reservation).

Reply:
If the Pindora call fails, I think the series will be "fixed" by the background task, so it should be fine to not fail the GQL mutation for better frontend experience 🤔 

(Also extra `remove` in comment `This might remove leave an empty series`)

Reply:
If the Pindora call fails there may be a non-transient problem with the API so that the task would also fail.

Reply:
Changed this to be aligned with how single reservation deny works